index.html

```diff
@@ -18,6 +18,7 @@
         <link rel="stylesheet" type="text/css" href="styles/css/portal.css"/>
         <link rel="stylesheet" type="text/css" href="styles/css/media.css"/>
         <link rel="stylesheet" type="text/css" href="styles/css/fix.css"/>
+        <link rel="stylesheet" type="text/css" href="styles/css/animations.css"/>
```

Comment:
this file has not been added

Reply:
I don't know why, I thought we had automatic `css` generation from `less` on deployment.